modules/ROOT/pages/create-or-modify-a-page.adoc

```diff
@@ -1,65 +1,178 @@
+
+== Create a fragment
+
+Use the same sequence of steps than to create a page from scratch.
+
+
+== Responsiveness
 
 You can optimize the page or form design for a given device type by configuring xref:widget-properties.adoc[device-specific values for the Width property] for widgets. Use the a device types bar in the Page editor to choose the target device type.
 image:images/images-6_0/pb-resolution.png[Device type selection]
 
-Custom widget is a special artifact that we will explain in a later chapter.
+== Modify an element
 
-== Preview a page, layout, form, or fragment
+After a form, page, layout, or fragment is created, you can update it by just clicking on the *_Pencil_* icon to open it in the editor. +
+Update the element by deleting or dragging and dropping widgets, business objects or fragments from the palette to the whiteboard. +
+Edit widget properties in the widget properties panel, create data, and bind them to the widgets.
 
-You can preview page, form and fragment rendering by clicking the *_Preview_* button. Another browser window opens with the artifact as it will be displayed after deployment. If you update the page, form or fragment, the preview is automatically refreshed when you save.
+== Preview an element
 
-You can display your page, layout, form or fragment with one of your theme application installed.
+You can preview page, form and fragment rendering by clicking the *_Preview_* button. Another browser window opens with the element as it will be displayed after deployment. If you update the page, form or fragment, the preview is automatically refreshed when you save. +
 
-From the preview window, you can also expand the preview outside of current preview mechanism, with the *Expand Preview in new window* button. This allows you to set new URL parameters, play with dynamic browser sizing, and to reuse same url in another browser to see how your form or page behaves in your user default browser.
+You can display your page, layout, form or fragment with one of your theme application installed. +
 
-You can use the a device types bar on the preview screen to choose the target device type. +
-The preview displays the artifact as it would be displayed on the selected type of device.
+From the preview window, you can also expand the preview outside of current preview mechanism, with the *Expand Preview in new window* button. This allows you to set new URL parameters, play with dynamic browser sizing, and to reuse same url in another browser to see how your form or page behaves in your user default browser. +
 
-Custom widget is a special artifact that we will explain in a later chapter.
+You can use the a device types bar on the preview screen to choose the target device type. +
+The preview displays the element as it would be displayed on the selected type of device. +
 
 [#export]
 
-== Export an artifact
+== Export an element
 
 You can export a page or a layout to deploy it in Bonita Portal as a custom page. +
```

Comment:
I suggest to explain that this way of deploying a page is only required on production when BCD is not available.

Reply:
done. I have also explained that exporting to import in Studio is only required when Git has not been set up.